src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -198,6 +198,7 @@ int sd_dhcp_server_new(sd_dhcp_server **ret, int ifindex) {
         server->address = htobe32(INADDR_ANY);
         server->netmask = htobe32(INADDR_ANY);
         server->ifindex = ifindex;
+        server->emit_router = 0;
```

Comment:
No, dhcp_server_emit_router is set to true in network_load_one, which leads to server->emit_router = 1 later on by default, so this does not matter.